src/main/steps/common/common.content.ts

```diff
@@ -111,6 +111,19 @@ const en = {
   apmCookiesHeadings: 'Allow cookies that measure website application performance monitoring?',
   useApmCookies: 'Use cookies that measure website application performance monitoring',
   doNotUseApmCookies: 'Do not use cookies that measure website application performance monitoring',
+  needInfoTitle: 'Why do we need this information and what will we do with it?',
```

Comment:
@ShakiraKP Why these labels are added in the base common content.ts file ?
These are specific to c100-rebuild, please remove from here & add in has-other-children content.ts

Reply:
As it was commonly used I thought it made sense to avoid duplicate code and its quite long but understood @channa-patil

Reply:
> @ShakiraKP Why these labels are added in the base common content.ts file ? These are specific to c100-rebuild, please remove from here & add in has-other-children content.ts

Good catch @channa-patil . Agree